api/tests/unit/domain/models/User_test.js

```diff
@@ -132,7 +154,44 @@ describe('Unit | Domain | Models | User', () => {
 
   });
 
-  describe('#email', function() {
+  describe('hasAccessToCertificationCenter', () => {
+
+    it('should be false is user has no access to no CertificationCenter', () => {
+      // given
+      const user = new User();
+      const certificationCenterId = 12345;
+
+      // when/then
+      expect(user.hasAccessToCertificationCenter(certificationCenterId)).to.be.false;
+    });
+
+    it('should be false is the user has access to many CertificationCenters, but not the one asked', () => {
```

Comment:
`should be false if user has access to many CertificationCenters, but not the given one`

Reply:
ok